web/controllers/posts_controller.ex

```diff
@@ -35,6 +36,7 @@ defmodule Blog.PostsController do
   end
 
   def update(conn, %{ "id" => id, "post" => post_params }) do
+    post_params = Dict.put(post_params, "author_id", current_user_id(conn))
```

Comment:
I believe this line could be removed, because after the post creation their relationship with the user will never change.
Just the `title` and `content` columns can be updated.